frontend/last-comments.ejs

```diff
@@ -25,7 +25,7 @@
     <script>
       var remark_config = {
         site_id: 'remark',
-        host: window.location.origin,
+        host: '<%= REMARK_URL %>' || window.location.origin,
```

Comment:
`REMARK_URL` is required param. It means this expression will always return the first value and we don't need to keep `window.location.origin`